Update 0-.ssh.md (#1139)
spectrocloud/librarium: content/docs/04-clusters/06-cluster-management/0-.ssh.md

import PointsOfInterest from 'shared/components/common/PointOfInterest';

# Overview
Palette supports SSH (Secure Shell) to establish, administer, and communicate with remote clusters. This section describes creating and managing SSH Keys in the Palette Management Console.

## Scope of your SSH Key

> Palette groups clusters for logical separation into **Projects**. Users and teams can be assigned roles within a project for granular control over permissions within the project scope. SSH key authentication is scoped to a project. Multiple users can be given access to a single project, however in order for them to gain access to clusters via SSH, they will need to have a public SSH key registered for each user.

Reviewer: ```suggestion
Palette groups clusters for logical separation into [Projects](/projects). Users and teams can be assigned roles within a project for granular control over permissions within the project scope. SSH key authentication is scoped to a project. Multiple users can be given access to a single project. To gain access to a cluster with SSH, you need to have a public SSH key registered in Palette.
```

> # Linking the Tenant SSH Key with Palette Management Console

Reviewer: ```suggestion
# Upload SSH Key in Palette

```

> Before you begin, generate an SSH key pair. We recommend using the RSA (Rivest-Shamir-Adleman) public-key cryptosystem to initiate a secure handshake between remote clusters for every cloud, data center, and edge environment supported by Palette.

Reviewer: ```suggestion
Before you begin, generate an SSH key pair. We recommend using the RSA (Rivest-Shamir-Adleman) public-key cryptographic system” to initiate a secure handshake between remote clusters for every cloud, data center, and edge environment supported by Palette.
```
Changing to the long form to help the reader.

> The steps to link the tenant SSH Key to Palette Management Console are as follows:

Reviewer: ```suggestion
The following steps guide you on how to upload your SSH key in Palette.
```

> 1. Log in to a Palette console, go to the left-hand side, and from the ribbon menu select **Project Settings**.

Reviewer: ```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).

```

> 2. From the Project Settings button, select the **SSH Keys** tab.

Reviewer: ```suggestion
2.  Navigate to the left **Main Menu** and select **Project Settings**. Next, select the **SSH Keys** tab.
```

> 3. Open the **+Add New SSH Key** tab to complete the Add Key form as below:

Reviewer: ```suggestion
3. Open the **+Add New SSH Key** tab to complete the **Add Key** form as below:
```
Using bold to indicate UI element.

	* **Name**: Provide a unique name for the SSH key.
	* **SSH Key**: Paste in the SSH public key contents from the key pair generated earlier.

> 4. Click **Confirm** to complete the wizard.

Reviewer: ```suggestion
4. Click **Confirm** to complete the wizard.

<br />
```

> **Note**: SSH keys can be edited and deleted later by using the kebab menu to the right of each key.

Reviewer: ```suggestion
<InfoBox>

SSH keys can be edited and deleted later by using the **three-dot Menu** to the right of each key.

</InfoBox>
```

> # Applying SSH keys

Reviewer: ```suggestion
# Use SSH Key

```
